src/rabbit_web_mqtt_handler.erl

```diff
@@ -74,10 +75,11 @@ init(Req, Opts) ->
 
 websocket_init(State = #state{conn_name = ConnStr, socket = Sock, peername = PeerAddr}) ->
     rabbit_log_connection:info("accepting Web MQTT connection ~p (~s)~n", [self(), ConnStr]),
+	rabbit_log_connection:info("rabbit_web_mqtt_connection_handler websocket_init"),
     AdapterInfo = amqp_connection:socket_adapter_info(Sock, {'Web MQTT', "N/A"}),
-    RealSocket = rabbit_net:unwrap_socket(Sock),
-    ProcessorState = rabbit_mqtt_processor:initial_state(Sock,
-        rabbit_mqtt_reader:ssl_login_name(RealSocket),
+  	%  RealSocket = rabbit_net:unwrap_socket(Sock),
+    ProcessorState = rabbit_mqtt_processor_custom:initial_state(Sock,
```

Comment:
I don't think this module exists in the MQTT plugin.

Reply:
Also, the name is not very specific. How exactly is "custom" different from a regular processor module? When should it be used? Why? The name raises a bunch of questions and answers none.